barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/toy_avm/toy_avm_circuit_builder.hpp

```diff
@@ -52,6 +52,12 @@ template <typename Flavor> class ToyAVMCircuitBuilder {
         const auto num_gates_log2 = static_cast<size_t>(numeric::get_msb64(num_gates));
         size_t num_gates_pow2 = 1UL << (num_gates_log2 + (1UL << num_gates_log2 == num_gates ? 0 : 1));
 
+        // We need at least 256 values for the range constraint
```

Comment:
why is this? do we have precomputed columns that have a minimum size of 256?

Reply:
It's just that in this implementation we add a table from 0-255 in compute_polynomials. It is not relation-generic, just for this concrete example.